File: NotificationSystem/NotificationSystem.Data/Classes/clsAdmin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using NotificationSystem.NotificationSystem.Data.NotificationSystemTableAdapters;

namespace NotificationSystem.NotificationSystem.Data.Classes
{
	public class clsAdmin
	{
		private string m_UserName = string.Empty;
		private string m_Password = string.Empty;
		private string m_Email = string.Empty;
		private string m_First = string.Empty;
		private string m_Last = string.Empty;

        public string UserName
        {
            get
            {
                return m_UserName;
            }
            set
            {
                m_UserName = value;
            }
        }

        public string Password
        {
            get
            {
                return m_Password;
            }
            set
            {
                m_Password = value;
            }
        }

        public string First
        {
            get
            {
                return m_First;
            }
            set
            {
                m_First = value;
            }
        }

        public string Last
        {
            get
            {
                return m_Last;
            }
            set
            {
                m_Last = value;
            }
        }

        public string Email
        {
            get
            {
                return m_Email;
            }
            set
            {
                m_Email = value;
            }
        }
    }
}

File: NotificationSystem/NotificationSystem.Data/Classes/clsAdmin.vb

Namespace NotificationSystem.NotificationSystem.Data.Classes
    Public Class clsAdmin
        Private m_UserName As String = String.Empty
        Private m_Password As String = String.Empty
        Private m_Email As String = String.Empty
        Private m_First As String = String.Empty
        Private m_Last As String = String.Empty

        Public Property UserName As String
            Get
                Return m_UserName
            End Get
            Set(value As String)
                m_UserName = value
            End Set
        End Property

        Public Property Password As String
            Get
                Return m_Password
            End Get
            Set(value As String)
                m_Password = value
            End Set
        End Property

        Public Property First As String
            Get
                Return m_First
            End Get
            Set(value As String)
                m_First = value
            End Set
        End Property

        Public Property Last As String
            Get
                Return m_Last
            End Get
            Set(value As String)
                m_Last = value
            End Set
        End Property

        Public Property Email As String
            Get
                Return m_Email
            End Get
            Set(value As String)
                m_Email = value
            End Set
        End Property
    End Class
End Namespace
